### PR TITLE
VDS-2341: Create dashboard component

### PR DESCRIPTION
**Pull request by Pav-el**
Description: 

--

QA-test: VDS-2343

Demo-test: VDS-2344

Download artifact URL: ghcr.io/virtocommerce/demo-xapi-app:1.0.0-pr-32-vds-2341-28497c59

### CODE REVIEW

File: src/app/components/dashboard-item/dashboard-item.component.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+})
+
+export class DashboardItemComponent {
+  @Input() itemLink =  '';
+
+  @Input() itemIconSrc =  '';
+
+  @Input() itemCaption =  '';
+
+  @Input() itemDescription =  '';
```
Comment (asvishnyakov) [inline src/app/components/dashboard-item/dashboard-item.component.ts]: Don't use prefix

File: src/app/components/dashboard/dashboard-content.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+export type DashboardSectionType = {
+  header: string;
+  iconSrc: string;
+  content: {
+      link: string;
+      caption: string;
+      description: string;
+  }[];
+};
```
Comment (asvishnyakov) [inline src/app/components/dashboard/dashboard-content.ts]: Use interface

File: src/app/components/dashboard/dashboard-content.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
+export const dashboardContent = [
```
Comment (asvishnyakov) [inline src/app/components/dashboard/dashboard-content.ts]: This variable is of `DashboardContentType`

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: src/app/services/checkout.service.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import { Component, OnDestroy } from '@angular/core';
-import { Router } from '@angular/router';
+import { Injectable } from '@angular/core';
+import { OnDestroy } from '@angular/core';
```
Comment (ilyawzrd) [inline src/app/services/checkout.service.ts]: Put imports in the same line
import { OnDestroy, Injectable } from '@angular/core';

**Review by ilyawzrd — CHANGES_REQUESTED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Review by ilyawzrd — APPROVED** [review]

**Pull request merged**